rest-api/src/main/resources/definitions/custom_activity_event_request.yml

```diff
@@ -0,0 +1,12 @@
+description: Request object for creating a Custom Activity Event.
+type: object
+required:
+    - eventKey
+properties:
+    eventKey:
```

Comment:
The main reason I created two objects is eventKey and eventId are slightly different, e.g. `eventA` vs `custom:eventA` and the appending behavior is done on the server side